docs/docs/contribution-juno.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title : Contribution to Juno
+---
+
+# Contributions Guidelines
+
+See [Engineering Guidelines](https://www.notion.so/Engineering-Guidelines-f916deb46ee242a0afaf36cc436e4c73) for more details
+
+## 1. Create a branch off of `main`
+
+Name the branch with a useful name. Ideally, read [this](https://stackoverflow.com/questions/273695/what-are-some-examples-of-commonly-used-practices-for-naming-git-branches) link.
+
+## 2. Writing code
+
+- Write the code you need to solve the issues:
+  - Ensure you have 100% coverage
+  - Ensure you add tests for the feature you add
+  - Follow this [commit guideline](https://www.conventionalcommits.org/en/v1.0.0/)
+  - Add documentation for code and functionality
+
+## 3. Create a pull request on Github
+
+Push your code to Github and create the pull request, link to the issue you are solving
+
+## 4. Submit for review
+
+- Assign the task to yourself in Github and select the appropiate reviewer
+- Assign the task in Notion to the appropriate reviewer
+
+# Adding new Issues
```

Comment:
Remove this and put in the header "Adding documentation" or something related to the docs because is not related to adding issues. Also an intro would be helpful because you jump straight away on creating a branch.